src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -248,7 +249,8 @@ private MessageConsumer newAcceptStream(
             final long applicationInitialId = begin.streamId();
             final long applicationRouteId = begin.routeId();
 
-            final SSLContext context = lookupContext.apply(store);
+            final StoreInfo storeInfo = lookupContext.apply(store);
+            final SSLContext context = storeInfo == null ? null : storeInfo.context;
```

Comment:
Seems like this one is really `lookupContext`, no need to let `StoreInfo` into `ClientStreamFactory`.